File: solutions/beecrowd/1035/1035.py
import sys

for line in sys.stdin:
    a, b, c, d = map(int, line.split())

    if b > c and d > a and c + d > a + b and c > 0 and d > 0 and a % 2 == 0:
        print('Valores aceitos')
    else:
        print('Valores nao aceitos')

File: solutions/beecrowd/1035/1035.clj
(ns main
  (:require [clojure.string :as str]))

(defn accepted-values? [a b c d]
  (and (> b c)
       (> d a)
       (> (+ c d) (+ a b))
       (> c 0)
       (> d 0)
       (zero? (mod a 2))))

(defn main []
  (loop [line (read-line)]
   (when line
     (let [[a b c d] (->> (str/split line #" ")
                          (map #(Integer/parseInt %)))]
       (if (accepted-values? a b c d)
         (printf "Valores aceitos%n")
         (printf "Valores nao aceitos%n"))
       (recur (read-line))))))

(main)
